modules/qna/src/views/full/LiteEditor.tsx

```diff
@@ -0,0 +1,181 @@
+import {
+  Breadcrumbs,
+  Button,
+  ControlGroup,
+  InputGroup,
+  Intent,
+  Menu,
+  MenuItem,
+  Popover,
+  Position
+} from '@blueprintjs/core'
+import { AccessControl } from 'botpress/utils'
+import React, { useEffect, useState } from 'react'
+
+import style from './style.scss'
+import Editor from './Editor'
+import { ExportButton } from './ExportButton'
+import { ImportModal } from './ImportModal'
+import Item from './Item'
+
+interface Props {
+  bp: any
+  topicName: string
+}
+
+interface QnaEntry {
+  id: string
+  data: any
+}
+
+export const LiteEditor = props => {
+  const [filter, setFilter] = useState('')
+  const [editId, setEditId] = useState('')
+  const [isEditing, setEditing] = useState(false)
+  const [data, setData] = useState<QnaEntry[]>()
+  const [importDialogOpen, setImportDialogOpen] = useState(false)
+
+  useEffect(() => {
+    // tslint:disable-next-line: no-floating-promises
+    fetchData()
+  }, [])
+
+  const getQueryParams = () => {
+    return {
+      params: {
+        categories: [props.topicName]
+      }
+    }
+  }
+
+  const fetchData = async () => {
+    const { data } = await props.bp.axios.get('/mod/qna/questions', getQueryParams())
+    setData(data.items)
+  }
+
+  const createNew = () => {
+    setEditId('')
+    setEditing(true)
+  }
+
+  const editItem = itemId => {
+    setEditId(itemId)
+    setEditing(true)
+  }
+
+  const cancelEditing = () => setEditing(false)
+
+  const deleteItem = async (id: string) => {
+    const needDelete = confirm('Do you want to delete the question?')
+
+    if (needDelete) {
+      const { data } = await props.bp.axios.post(`/mod/qna/questions/${id}/delete`, getQueryParams())
+      setData(data.items)
+    }
+  }
+
+  const toggleEnableItem = async (item: any, id: string, isChecked: boolean) => {
+    item.enabled = isChecked
+
+    const { data } = await props.bp.axios.post(`/mod/qna/questions/${id}`, item, getQueryParams())
+    setData(data.items)
+  }
+
+  const categories = [{ label: props.topicName, value: props.topicName }]
+
+  return (
+    <div>
+      {!isEditing && (
+        <div className={style.liteHeader}>
+          <div className={style.liteSearch}>
+            <ControlGroup>
+              <InputGroup
+                id="input-search"
+                placeholder="Search for a question"
+                tabIndex={1}
+                value={filter}
+                onChange={e => setFilter(e.currentTarget.value)}
+              />
+
+              <AccessControl resource="module.qna" operation="write">
+                <Button
+                  id="btn-create-qna"
+                  text="Add new"
+                  icon="add"
+                  style={{ marginLeft: 20 }}
+                  intent={Intent.PRIMARY}
+                  onClick={() => createNew()}
+                />
+              </AccessControl>
+            </ControlGroup>
+          </div>
+
+          {/*
+          TODO: Support for import/export scoped to a category
```

Comment:
Your export button already does just that no ?

Reply:
It works for the full view, but it doesn't support category-scoped at the moment (only the whole qna questions)